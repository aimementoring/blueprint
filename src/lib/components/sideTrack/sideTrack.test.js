import React from 'react';
import renderer from 'react-test-renderer';
import SideTrack from './sideTrack.js';

describe('SideTrack', () => {
  it('renders properly', () => {
    const sideTrackParagraph = (
      <div>
        Testing paragraph with a <a href="#">Link</a> inside
      </div>
    );
    const tree = renderer
      .create(
        <SideTrack title="Side track title" paragraph={sideTrackParagraph} position="right" />,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
