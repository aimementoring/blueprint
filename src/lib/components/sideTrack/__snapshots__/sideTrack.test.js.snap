// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SideTrack renders properly 1`] = `
<div
  className="theme-purple"
>
  <div
    className=""
  >
    <div
      className="sideTrackCallOut right "
    >
      <div
        className="sideTrackCallOutWrapper"
      >
        <div
          className="sideTrackText"
        >
          <h3
            className="sideTrackTitle"
          >
            Side track title
          </h3>
          <p>
            <div>
              Testing paragraph with a 
              <a
                href="#"
              >
                Link
              </a>
               inside
            </div>
          </p>
        </div>
        <div
          className="emojiWrap left"
        >
          <span
            aria-label="point down"
            role="img"
          >
            👇🏿
          </span>
        </div>
      </div>
    </div>
  </div>
</div>
`;
